.github/workflows/universalnrp-test.yml

```diff
@@ -44,25 +44,4 @@ jobs:
       resource-count: ${{ matrix.policy-package.resource-count }}
       skip-remediation: ${{ matrix.skip-remediation }}
       install-osconfig: ${{ matrix.install-osconfig }}
-      tag: ${{ matrix.target.tag }}
-
-  report:
```

Comment:
Why is this being removed? This is what consumes the test report

Reply:
It is necessary, notice the hash for the commit is not updated with the latest hash
![image](https://github.com/Azure/azure-osconfig/assets/224804/35a7463a-f15b-4e71-bc7f-79cde1e52eb1)

Reply:
We should probably keep some form of a test report, odd this one is not updating to the latest commit though. I see the edits made by the bot but latest is not reflected.